client/pubspec.yaml

```diff
@@ -1,26 +1,93 @@
 name: client
-description: Community portal frontend
-repository: https://github.com/openfoodfacts/community-portal
+description: A new Flutter project.
```

Comment:
Why did you change the description and URL?

Reply:
By mistake it got pushed.